quantum/split_common/transport.c

```diff
@@ -147,6 +192,12 @@ bool transport_master(matrix_row_t matrix[]) {
   }
 #  endif
 
+#  ifdef ENCODER_ENABLE
+  static int8_t sencoder[NUMBER_OF_ENCODERS];
+  memcpy((void*)(sencoder), (const void *)serial_s2m_buffer.sencoder, sizeof(sencoder) );
+  encoder_update_raw(&sencoder[0]);
```

Comment:
The separate array and memcpy is unnecessary, just cast serial_s2m_buffer.sencoder appropriately and pass directly to encoder_update_raw. It doesn't matter that it's volatile.

Reply:
fixed